GameData/RP-1/Contracts/Earth Satellites/FirstMolniyaSat.cfg

```diff
@@ -7,7 +7,7 @@ CONTRACT_TYPE
 	
 	tag = exclude_EarlySatellite
 
-	description = <b>Program: Targeted Satellites<br>Type: <color=green>Required</color></b><br><br>A Molniya orbit is a type of highly elliptical orbit with an inclination of 63.4 degrees, an argument of perigee of -90 degrees, and an orbital period of one half of a sidereal day. Place a satellite into a Molniya orbit.&br;&br;There will be an orbit showing in the map view and tracking station. You do not have to place the satellite into this orbit, but it is instead meant to show you where your perigee and apogee should approximately be located, but more importantly, it shows the proper shape of the orbit.
+	description = <b>Program: Early Commercial Applications<b> or <b>Targeted Satellites<br>Type: <color=green>Required</color></b><br><br>A Molniya orbit is a type of highly elliptical orbit with an inclination of 63.4 degrees, an argument of perigee of -90 degrees, and an orbital period of one half of a sidereal day. Place a satellite into a Molniya orbit.&br;&br;There will be an orbit showing in the map view and tracking station. You do not have to place the satellite into this orbit, but it is instead meant to show you where your perigee and apogee should approximately be located, but more importantly, it shows the proper shape of the orbit.
```

Comment:
But CA now includes its own FirstMolniyaSat ...?

Reply:
Oh, I see, added that before I created the new one.